src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let marriageStatus = 'I am not married';
+
+  if (person.isMarried) {
+    const partner = person.sex === 'f' ? 'husband' : 'wife';
+
+    marriageStatus = `${person.partnerName} is my ${partner}`;
+  }
```

Comment:
You can write a function that returns your correct text. Or you can write it inside JSX ( worse readability). Using let in global components scope worse for code support.

Reply:
Got it!
I`m not sure how to write it properly here
Could you please review my new solution?
Thanks!